app/src/main/res/values/arrays.xml

```diff
@@ -7,6 +7,7 @@
         <item>Wifi Wixel + BT Wixel</item>
         <item>G4 Share Receiver</item>
         <item>G5 Transmitter</item>
+        <item>LimiTTer</item>
```

Comment:
1) Yes it does. Like the firmware for the wixel (wixel-xDrip or xBridge2) it has a firmware for the microcontroller that operates the NFC reader and the HM11.
2) It uses calibrations but at the moment just to set the offset (intercept). Once there is more data, we might adjust the slope as well (very likely, but not in as wide a range as for Dexcom sensors).